intro-HPC/ch_apptainer.tex

```diff
@@ -31,34 +31,31 @@ \section{Restrictions on image location}
 
 \section{Available filesystems}
 
-All HPC-UGent shared filesystems will be readily available in a Singularity container,
+All HPC-UGent shared filesystems will be readily available in a Apptainer/Singularity container,
 including the home, data and scratch filesystems, and they will be accessible via the
 familiar \lstinline|$VSC_HOME|, \lstinline|$VSC_DATA*| and \lstinline|$VSC_SCRATCH*| environment variables.
 
 \section{Singularity Images}
 
 \subsection{Creating Singularity images}
 
-Creating new Singularity images or converting Docker images, by default, requires admin privileges,
+Creating new Apptainer/Singularity images or converting Docker images, by default, requires admin privileges,
 which is obviously not available on the \hpcInfra infrastructure. However,
-if you use the \lstinline|--fakeroot| option, you can make new Singularity images or convert 
+if you use the \lstinline|--fakeroot| option, you can make new Apptainer/Singularity images or convert 
 Docker images. 
 
-When you make Singularity or convert Docker images you have some restrictions.
-
-\begin{itemize}
-    \item You can only create Singularity images on RHEL 8 machines. The current RHEL 8 systems are
-        \lstinline|gligar08| login node, \lstinline|doduo| cluster, and \lstinline|slaking| cluster.
-        The created images can be used on any cluster.
-    \item Due to the nature of \lstinline|--fakeroot| option, we recommend to write your singularity 
-        image to a globally writable location, like \lstinline|/tmp|, or \lstinline|/local| directories. 
-        Once the images is created, you should move it to your desired destination.
-\end{itemize}
+Due to the nature of \lstinline|--fakeroot| option, we recommend to write your apptainer 
+image to a globally writable location, like \lstinline|/tmp|, or \lstinline|/local| directories. 
+Once the images is created, you should move it to your desired destination.
+An example to make a apptainer container:
+\begin{prompt}
+%\shellcmd{APPTAINER_CACHEDIR=/tmp/ APPTAINER_TMPDIR=/tmp/ apptainer build --fakeroot /tmp/tensorflow-21.10-tf1-py3.sif docker://nvcr.io/nvidia/tensorflow:21.10-tf1-py3}%
```

Comment:
I'm no latex expert either. maybe @alvarosimon can help. but is the `-` a special char?

Reply:
@hajgato this should work:

```
%\shellcmd{APPTAINER\_CACHEDIR=/tmp/ APPTAINER\_TMPDIR=/tmp/ apptainer build --fakeroot /tmp/tensorflow-21.10-tf1-py3.sif docker\://nvcr.io/nvidia/tensorflow\:21.10-tf1-py3}%
```

Reply:
fixed it